onboarding/src/Components/Dashboard/DashboardPage.js

```diff
@@ -0,0 +1,117 @@
+import React, { useRef, useState, useEffect } from "react";
```

Comment:
niby wiem, że to działa, ale jakoś tak dziwnie wygląda ten import

Reply:
I know that multi lines are prettier :) ,
but this is quite common.
I mean default exported and other between brackets.
It's even that you can not import something not default outside curly brackets -
- I experienced this in a hard way :/